x/tokenomics/keeper/token_logic_modules_test.go

```diff
@@ -979,6 +1246,74 @@ func getNumTokensClaimed(
 	return numTokensClaimedRat.Num().Int64() / numTokensClaimedRat.Denom().Int64()
 }
 
+// rewardDistribution tracks rewards distributed to validators and delegators
+type rewardDistribution struct {
```

Comment:
Seems like an important time injected in the middle of a hueg file.

Can you find a better place for it?